packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.types.ts

```diff
@@ -32,44 +32,45 @@ export interface ICommandBarProps extends React.HTMLAttributes<HTMLDivElement> {
   componentRef?: IRefObject<ICommandBar>;
 
   /**
-   * Items to render. ICommandBarItemProps extend IContextualMenuItem
+   * Items to render. ICommandBarItemProps extends IContextualMenuItem.

```

Comment:
Comment changes only, aside from the one prop that was `Boolean` for some reason instead of `boolean`